src/Backend/SgateShopgatePlugin/Models/Export/Product.php

```diff
@@ -809,6 +809,14 @@ public function prepareProperties($article, $detail)
             }
         }
 
+        foreach ($properties as &$values) {
+            foreach ($values as &$value) {
+                if ($value instanceof DateTime) {
+                    $value = date_format($value, 'Y-m-d');
```

Comment:
Gibt es bei Shopware eine Konfiguration für dieses Format pro StoreView/Sprache oder werden DateTimes auch statisch in diesem Format  im Frontend ausgegeben? Wird DateTime in Shopware nur für Dates benutzt oder droppen wir hier bewusst den Time Part?